translations/get5.phrases.txt

```diff
@@ -0,0 +1,238 @@
+"Phrases"
+{
+    // get5.sp
+    "ReadyToVetoInfoMessage"
+    {
+        "en"            "Type {GREEN}!ready {NORMAL}when your team is ready to veto."
+    }
+    "WaitingForCastersReadyInfoMessage"
+    {
+        "en"            "Waiting for the casters to type {GREEN}!ready {NORMAL}to begin."
+    }
+    "ReadyToRestoreBackupInfoMessage"
+    {
+        "en"            "Type {GREEN}!ready {NORMAL}when your team is ready to restore the match backup."
+    }
+    "ReadyToKnifeInfoMessage"
+    {
+        "en"            "Type {GREEN}!ready {NORMAL}when your team is ready to knife."
+    }
+    "ReadyToStartInfoMessage"
+    {
+        "en"            "Type {GREEN}!ready {NORMAL}when your team is ready to begin."
+    }
+    "WaitingForEnemySwapInfoMessage"
+    {
+        "en"            "%s won the knife round. Waiting for them to type !stay or !swap."
+    }
+    "WaitingForGOTVBrodcastEndingInfoMessage"
+    {
+        "en"            "The map will change once the GOTV broadcast has ended."
+    }
+	"NoMatchSetupInfoMessage"
+	{
+		"en"			"There is no match setup"
+	}
+    "YourAreNotAPlayerInfoMessage"
+    {
+        "en"            "You are not a player in this match"
+    }
+    "TeamIsFullInfoMessage"
+    {
+        "en"            "Your team is full"
+    }
+    "TeamForfeitInfoMessage"
+    {
+        "en"            "%s failed to ready up in time and has forfeit."
```

Comment:
Unless sourcemod has changed since I last worked with it (I don't think it has), you have to specify arguments differently than the printf style used here: https://wiki.alliedmods.net/Translations_(SourceMod_Scripting)#File_Format

So "TeamForfeitInfoMessage" would need a ``"#format {1:s}"`` line and use ``{1}`` instead of a ``%s`` in the translation string itself.